src/Utils/handler.js

```diff
@@ -0,0 +1,38 @@
+import DiscountCalculator from '../Domain/DiscountCalculator.js';
+import InputView from '../View/InputView.js';
+import OrderProcessor from '../Domain/OrderProcessor.js';
+import OutputView from '../View/OutputView.js';
+
+const ErrorHandlerAndRetry = async (handler, retry) => {
```

Comment:
camelCase로 작성하는게 바람직해보입니다!

```suggestion
const errorHandlerAndRetry = async (handler, retry) => {
```